lib/govuk_app_config/govuk_data_sync.rb

```diff
@@ -0,0 +1,31 @@
+require "time"
+
+class GovukDataSync
+  def initialize(govuk_data_sync_period)
+    @from, @to = false
+    begin
+      parts = govuk_data_sync_period&.split("-")
+      @from, @to = parts.map { |time| Time.parse(time) } if parts&.count == 2
+    rescue ArgumentError
+      # At least one of the parts was malformed. Leave @from/@to as false.
+    end
+  end
+
+  def in_progress?
+    @from && @to && in_time_range?(@from, @to)
+  end
+
+private
+
+  def in_time_range?(from, to)
+    hour_is_in_range = Time.now.hour >= from.hour || Time.now.hour <= to.hour
+    minute_is_in_range = if Time.now.hour == from.hour
+                           Time.now.min >= from.min
+                         elsif Time.now.hour == to.hour
+                           Time.now.min <= to.min
+                         else
+                           true
+                         end
```

Comment:
A thing to note is that this will be set to whatever timezone the software running is in, which would be UTC for us. Is that consistent with puppet? (I assume it is)

It might be worth adding a comment to note that the times are expected to be UTC?